File: NirvanaSMS.Api.CSharp/Core/Base/UserBase.cs
using NirvanaSMS.Api.Core.Interface;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NirvanaSMS.Api.Core.Base
{
    public class UserBase : IUser
    {
        public string UserName { get; set; }
        public string ApiPassword { get; set; }
    }
}

File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/base/UserBase.java
package org.nirvanasms.api.core.base;

import org.nirvanasms.api.core.interfaces.IUser;

public class UserBase implements IUser {
	private String userName;
	private String apiPassword;
	
	public String getUserName() {
		return userName;
	}
	public void setUserName(String userName) {
		this.userName = userName;
	}
	public String getApiPassword() {
		return apiPassword;
	}
	public void setApiPassword(String apiPassword) {
		this.apiPassword = apiPassword;
	}
	
}
